script.js

```diff
@@ -0,0 +1,11 @@
+function addNum()
+{
+ var num1, num2, num3    
+ num1=parseInt(document.addCalc.txt1.value);
+ num2=parseInt(document.addCalc.txt2.value);
+ num3=num1+num2;
```

Comment:
Generally we'd do variable creation and assignment on the same line, makes it more readable.

```JavaScript
var num1 = parseInt(document.addCalc.txt1.value)
var num2 = parseInt(document.addCalc.txt2.value)
var num3 = num1 + num2
```